feature/profile/src/main/java/com/yapp/feature/profile/component/ProfileInformationSection.kt

```diff
@@ -0,0 +1,40 @@
+package com.yapp.feature.profile.component
+
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.size
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.unit.dp
+import com.yapp.core.designsystem.component.chip.ChipColorType
+import com.yapp.core.designsystem.component.chip.YappChipLarge
+import com.yapp.core.designsystem.theme.YappTheme
+
+@Composable
+internal fun ProfileInformationSection(
+    modifier: Modifier = Modifier,
+    userName: String,
+    userGeneration: String,
+    userPosition: String,
+    userRole: String
+) {
+    Row(
+        modifier = modifier.padding(20.dp),
+        horizontalArrangement = Arrangement.spacedBy(16.dp),
+        verticalAlignment = Alignment.CenterVertically
+    ) {
+        Box(modifier = Modifier.size(50.dp))
+        Column {
+            Row(verticalAlignment = Alignment.CenterVertically, horizontalArrangement = Arrangement.spacedBy(8.dp)) {
+                Text(text = userName, style = YappTheme.typography.title2Bold)
+                YappChipLarge(text = userRole, colorType = ChipColorType.Main, isFill = true)
```

Comment:
```suggestion
                YappChipSmall(text = userRole, colorType = ChipColorType.Main, isFill = true)
```

R: radius가 8px이기 때문에 Small로 사용해주세요

Reply:
[a0b7eb8](https://github.com/YAPP-admin/yappu-world-android/pull/130/commits/a0b7eb820b2d760102aef6b840625cff7736ea0a)

변경처리 했습니다